server/ngwmn/templates/site_location.html

```diff
@@ -26,6 +26,17 @@ <h1>{{ feature.SITE_NAME }}</h1>
                     Summary
                 </button>
                 <div id="ts-a1" class="usa-accordion-content">
+                    <div>
+                        <h4>Description</h4>
+                        <p>
+                            This site is a {{ _(feature.SITE_TYPE) }}. It is named "{{ _(feature.SITE_NAME) }}"
```

Comment:
Why  is the template variable {{ _(feature.SITE_TYPE) }} rather than just {{ feature.SITE_TYPE }}?

Reply:
`_` is a helper macro in this template that puts in `-` as a placeholder, when the string is empty or None.

But yeah in this case if the field is nullable, it seems it may make sense to wrap the sentence in an if condition, to omit it if it's not set.

Or perhaps choose an alternate default value (the second parameter to `_`) that makes sense given the context... like `This site information was provided by {{ _(feature.AGENCY_NM, 'an unspecified agency.') }}`

Reply:
Why not use the default filter instead? I think the intent would be clearer.

Reply:
I suppose it's debatable, but I made it a macro because it's repeated many times.

Reply:
Historically we were asked to use "unknown" for missing values.

Reply:
Historically for NGWMN? I think Candice is the one to ask what she wants to do if values are missing.

Reply:
Just to add my two cents: since the one of the 'Internet of Water' goals is to make water data more accessible and understandable to the average person, I think we should avoid creating statements that have no content value such as 'This site is a unknown' or 'This site is a unspecified . . .' These sentences add no information and consume the user's time. I vote for omitting sentences that don't add value for the user.

Reply:
@abriggs-usgs are you recommending that this be rejected in favor of something else? I am okay with closing it without merging. Please know that this not my wording.

There was agreement within the team about the usefulness of this content. When this story was created the comments were something to this effect. "We need a description on the location pages and there is text on the current/legacy pages that will do for now. It is not the best description but it will serve as a place holder."

Reply:
No, I am just making a suggestion. Something that could be discussed with Candice. It would up to you and Candice if you used the suggestion in this task, a later one, or not at all.